docs/it-manual/sre-playbook/disaster-recovery.md

```diff
@@ -0,0 +1,118 @@
+  {% endhint %}
+
+- Set `POSTGRESQL_VERSION` to the version of the existing database.
+  
+  - To find the postgres version go to the AWS console and navigate to RDS.
+  - Click on the database you will be restoring to get the details page
+  - Click on the "Configuration" tab
+  - Look for "Engine version" to get the value you for `POSTGRESQL_VERSION`.
+
+  {% hint style="warning" %}
+  Upgrading Postgres versions should NOT be done at the same time as restoring an environment. Restoring a snapshot of the old database to a new database with differing Postgres versions increases the risk of failure and compatibility.
+  {% endhint %}
+
+- Set POSTGRES_RESTORE_SNAPSHOT_IDENTIFIER to the snapshot to restore. 
+  
+  - To find the snapshot identifier go to the AWS console and navigate to RDS > Snapshots.
+  - Use the snapshot identifier (not the name, the identifier begins with `rds:`) for the value for `POSTGRES_RESTORE_SNAPSHOT_IDENTIFIER`.
+
+Do NOT change any other settings.
+
+{% hint style="danger" %}
+The `CIVIFORM_VERSION` MUST be the same version that was in use with the selected snapshot. Upgrading the version of CiviForm should NOT be done at the same time as restoring an environment.
+{% endhint %}
+
+## Initialize the deployment
+
+Verify that deployment tool is correctly setup by running `run bin/doctor --config=NEWCONFIGFILE`
+
+## Restore into new environment
+
+{% hint style="warning" %}
+This process will take some time to run. Depending upon the size of the database being restored expect on average 30-60 minutes for completion. There will be large periods where you need to wait while the setup runs, but there are also points where you will be prompted for values.
+{% endhint %}
+
+- To begin the restoration run `bin/setup --config=NEWCONFIGFILE`.
+- Wait as setup runs
+- When prompted enter the values for all the requested secrets.
+- When prompted for Postgres snapshot restore follow all prompts. You will be asked for the app prefix of the old environment. This is the value from `APP_PREFIX` of the original config file.
+- Wait as setup runs
+
+When complete, make note of the endpoint url listed at the end of the deployment. It will be the one that has the `APP_PREFIX` from your new config file and ends with "elb.amazonaws.com".
+
+At this point the new environment is setup and should be online at the above endpoint url if no errors occurred during this process.
+
+## Verify the new site is working
+
+Test that the site responds by using the endpoint url from the previous section.
+
+- Paste the endpoint url into your browser, you will see an invalid SSL certificate error and may not be able to process depending on your browser settings. If you can temporarily accept the invalid certificate you should see the CiviForm program page load.
+- Alternatively, you can run `curl --insecure --location PUT_ENDPOINT_URL_HERE`. The results should be HTML.
+
+## Change DNS settings to point to new site
```

Comment:
Will this vary depending on the DNS provider? Or will anyone doing this process need to update these settings in AWS?

Reply:
@rockycodes I do not have an answer to this. Seattle delegates specific subdomains to Route53 to manage and that's the only way I know how to manage this.

Reply:
gotcha, i think the answer then is we should try this out. @nb1701 or @dkatzz do you know if this is something all deployments would need to do? for our test deployments i figure there's a step to update the lb url in cloudflare, but i'm not sure if we would also need to do this in AWS.